compose/foundation/foundation/src/desktopMain/kotlin/androidx/compose/foundation/TooltipArea.desktop.kt

```diff
@@ -135,19 +135,21 @@ fun TooltipArea(
             .onGloballyPositioned { parentBounds = it.boundsInWindow() }
             .onPointerEvent(PointerEventType.Enter) {
                 cursorPosition = it.position
-                startShowing()
+                if (!isVisible && !it.buttons.areAnyPressed) {
+                    startShowing()
+                }
```

Comment:
This fixes the case of press -> exit -> enter
Previously we would re-show the tooltip.